ajax/libs/prettycron/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "prettycron",
+  "version": "0.10.0",
+  "description": "Display cron runspecs in human-readable format",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/azza-bazoo/prettycron.git"
+  },
+  "author": "Hourann Bosci (http://hourann.com)",
+  "license": "LGPL",
+  "npmName": "prettycron",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
```

Comment:
What's the reason why to include them all?
We don't even need `LICENSE.md`, `node_modules/`, `package.json`, `README.md` and `test/`.